### DISCUSSION TITLE
Artigo sobre Layout Shift e CLS

### DISCUSSION THREAD

**Discussion by tarciozemel** [frontendbr/forum#1792] [Dicas]
Assunto relativamente recente, mas ainda não vi brasucas discutindo muito a respeito: Layout Shift e CLS. 

Minha contribuição inicial é a tradução do excelente artigo de Addy Osmani "Optimize Cumulative Layout Shift": [Deslocamento de Layout (Layout Shift) e Cumulative Layout Shift (CLS): guia completo](https://desenvolvimentoparaweb.com/ux/deslocamento-layout-shift-cumulative-layout-shift-cls/).

E aí, quem já tá levando isso em consideração como métrica de qualidade nos projetos?

**Comment by lucianobarauna** [comment]
Eu vi o link do artigo original em alguma newsletter e achei bem interessante. Acho que quando chegar essa propriedade `aspect-ratio` vamos conseguir fazer isso de uma forma responsiva. 

Temos uma meta para criar um lazzyload na loja virtual que trampo hoje.

Bela iniciativa para a tradução.

**Comment by felipefialho** [comment]
Que artigo foda @tarciozemel. Obrigado por compartilhar.

**Comment by tarciozemel** [comment]
@lucianobarauna Por falar nisso, ó: https://css-tricks.com/a-first-look-at-aspect-ratio/

@felipefialho Eu que agradeço, Felipe. Vi seu comment lá. ^^

**Comment by lucianobarauna** [comment]
@tarciozemel eu vi no twitter deles também tá na lista para ler :-)
Acho que vai ser uma ótima propriedade para o CSS
